modules/data-loading/pages/kafka-loader/index.adoc

```diff
@@ -29,7 +29,6 @@ A resumed job picks up from where it was stopped before and will not consume mes
 ====
 
 == Supported file formats
-* Avro
```

Comment:
Keep Avro, but add a note with a hyperlink that says something like
"Avro (when sourced from an external Kafka cluster using MirrorMaker)"

Reply:
Thank you I have added.